recipe/conda_build_config.yaml

```diff
@@ -3,3 +3,6 @@
 # mkl_fft and mkl_random, and then numpy.
 # If only_build_numpy_base: yes, build numpy-base only; otherwise build all the outputs.
 only_build_numpy_base: no
+
+mkl:
+  - 2021.*
```

Comment:
If we pin to MKL 2021, this updated numpy will not be usable with anything that's built with our current aggregate cbc.yaml. We should do one of:
* Rollback the MKL pin the aggregate cbc.yaml;
* Figure out what's broken/blocking the build with MKL 2022; or
* Build for both MKL 2021 and 2022.

Reply:
Hi Cheng, the pin is updated on cbc but nothing has been rebuilt against MKL 2022 so far (such as mkl-service, which is a dependency of numpy).
The idea was to plan this for after the AD release. There is also a spike ticket to align numpy base versions to 1.22.

If using the recipe cbc is not acceptable, I think we should rollback the pin on aggregate cbc.